src/stylesheets/components/_concept.scss

```diff
@@ -0,0 +1,37 @@
+.concept {
+  background: linear-gradient($florence-blue, $florence-light-blue);
+  color: $white;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-top: 180px;
+  padding-top: 130px;
+  position: relative;
+  width: 100%;
+  
+  .section-title {
+  	color: $white;
+  	margin-bottom:-10px;
```

Comment:
Indents are off here (too much). I wonder if stylelint isn't running? Might need to add it as an option to your blendid config. I'll look into it some and see what I can find for you.

Reply:
According to the stylelint docs, there are a 5 different ways to use it, 2 of which are:
* via a plugin for your build tool
  * Digging into blendid a little bit, I don't think it currently supports passing additional plugins directly to the stylesheets task. So you'd have to write a new css processing task (including the gulp-stylelint plugin) and pass it in the task config in the `alternateTask` option. This doesn't sound great, so the next best options is...
* via a plugin for your text editor
  * Just install http://www.sublimelinter.com/en/stable/ and have it work directly in your editor. At first glance it looks tricky to set up, but I've never used it so I can't vouch for it.

Reply:
Ah, this is because I switched to Sublime from Atom and didn't do that install yet - on it now.

Reply:
The docs I referenced: https://github.com/stylelint/stylelint#getting-started